.rubocop_todo.yml

```diff
@@ -176,7 +184,18 @@ Naming/MethodName:
 Naming/PredicateName:
   Enabled: false
 
-# Offense count: 144
+# Offense count: 60
+# Cop supports --auto-correct.
+# Configuration parameters: PreferredName.
+Naming/RescuedExceptionsVariableName:
+  Enabled: false
+
+# Offense count: 1
```

Comment:
Would it be easier to fix this particular case or disable the cop inline?

Reply:
I'd like to fix this after the merge.

Reply:
GH-6887